src/components/AffectedOfferingForm.vue

```diff
@@ -23,29 +23,32 @@ const isScreenSortaSmall = computed(() => screenWidth.value < 950);
 
 const modelValue = defineModel<ZodAffectType>({ default: null });
 
-function getCvssData(issuer: string, version: string) {
-  return modelValue.value.cvss_scores.find(
-    (assessment) => assessment.issuer === issuer && assessment.cvss_version === version
-  );
-}
-
-if (!getCvssData('RH', 'V3')) {
-  modelValue.value?.cvss_scores?.push(
-    {
-      vector: null,
-      uuid: null,
-      issuer: 'RH',
-      cvss_version: 'V3',
-      comment: 'hardcoded comment', // TODO: remove this line when the comment field is added to the UI
-      created_dt: null,
-      score: null,
-      embargoed: modelValue.value.embargoed,
-    } as ZodAffectCVSSType
-  );
-}
-
-// Type assertion to prevent TS error in template
-const affectCvssScore = modelValue.value.cvss_scores.find(({ issuer }) => issuer === 'RH') as ZodAffectCVSSType;
+// function getCvssData(issuer: string, version: string) {
+//   return modelValue.value.cvss_scores.find(
+//     (assessment) => assessment.issuer === issuer && assessment.cvss_version === version
+//   );
+// }
+
+// if (!getCvssData('RH', 'V3')) {
+//   modelValue.value?.cvss_scores?.push(
+//     {
+//       vector: null,
+//       uuid: null,
+//       issuer: 'RH',
+//       cvss_version: 'V3',
+//       comment: 'hardcoded comment', // TODO: remove this line when the comment field is added to the UI
+//       created_dt: null,
+//       score: null,
+//       embargoed: modelValue.value.embargoed,
+//     } as ZodAffectCVSSType
+//   );
+// }
+
+const affectCvssVector = computed(
+  () => modelValue.value.cvss_scores.find(({ issuer }) => issuer === 'RH')
```

Comment:
This won't differentiate between versions. I think currently majority of Affect CVSS are V3, but if there will be any V2 or in the future V4, this will break and might not show the right data.